### DISCUSSION TITLE
SkullOwner type?

### DISCUSSION THREAD

**Discussion by rudde0** [ShaneBeee/SkBee#589] [Q&A] [answered]
Since SkBee 3.4.1, we need to specify the nbttype of tags. In order this I need to change a lot of code.

My question is related with SkullOwner.

Please assume that I'm holding a skinned player head on my hand. Before the update I was simply doing this,
```
set {head} to (player head with nbt compound from "{SkullOwner:%tag "SkullOwner" of nbt compound of (player's held item)%}"
```

But in new update, we need to specify the nbttype of SkullOwner's tag. I tried things such as
- `compound list tag "SkullOwner" of nbt compound of (player's held item)` (returns null)
- `string tag "SkullOwner" of nbt compound of (player's held item)` (returns "") (blank)
but I exactly need the output of `tag "SkullOwner" of nbt compound of (player's held item)`. How can I provide this?

**Comment by rudde0** [comment]
One more thing,

I paste data of villager with this code:
`set tag "VillagerCustom" of nbt compound of (player's held item) to (nbt compound from (last spawned villager))`

and when I need it I paste the villager's data with this:
`add (tag "VillagerCustom" of nbt compound of (player's held item)) to (nbt compound of (last spawned villager))`

Which types should I need to use?

**Comment by ShaneBeee** [comment, marked as answer]
If you have trouble finding the type of tag, you have a few options:

1) Use the tag type expression https://skripthub.net/docs/?id=10695

2) You can check McWiki, it will tell you exactly the tag type:
<img width="522" alt="Screenshot 2024-03-13 at 8 46 42 AM" src="https://github.com/ShaneBeee/SkBee/assets/20008482/3aa2ecc9-5fb4-4e20-aa48-70f4df7dc886">

You can also check out the [**Data Types Wiki**](https://skripthub.net/docs/?id=10695) or the [**Data Types McWiki**](https://minecraft.wiki/w/NBT_format#Data_types) for further information.
